contracts/Bridge.sol

```diff
@@ -15,7 +15,7 @@ contract Bridge {
 
     enum Vote {No, Yes}
     enum RelayerThresholdProposalStatus {Inactive, Active}
-    enum DepositProposalStatus {Inactive, Active, Denied, Passed, Transferred}
+    enum DepositProposalStatus {Inactive, Active, Passed, Transferred}
```

Comment:
why are we removing the Denied status? Is it replaced by Inactive if a proposal is voted out?

Reply:
Yes